src/components/card/FlipCard.js

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from 'styled-components/macro'
+import { useState } from 'react'
+import mainBackground from '../../assets/images/mainBackground.jpg'
+
+export default function FlipCard(props) {
+  const [isCardFlipped, setIsCardFlipped] = useState(false)
+  const isFreeOfCharge = props.freeOfCharge
+  const handleFlipCardClick = () => {
+    setIsCardFlipped(!isCardFlipped)
+  }
+  return (
+    <Card onClick={handleFlipCardClick}>
+      <CardContent isCardFlipped={isCardFlipped}>
+        <CardFront>
+          <CardTitle>{props.name}</CardTitle>
+          <CardSubtitle>{props.city}</CardSubtitle>
+        </CardFront>
+        <CardBack>
+          <CardBody>
+            <p>
+              <b>Name:</b> {props.name}
+            </p>
+            <br />
+            <p>
+              <b>Description:</b> {props.description}
+            </p>
+            <br />
+            <p>
+              <b>Address:</b> {props.street}
+              <br />
+              {props.zipCode} {props.city}, {props.country}
+            </p>
+            <br />
+            <p>
+              <b>Opening Hours:</b> {props.openingHours}
+            </p>
+            <br />
+            <p>
+              <b>Website:</b>{' '}
+              <a
+                href={props.website}
+                target="_blank"
+                rel="noreferrer"
+                onClick={event => event.stopPropagation()}
+              >
+                Click here!
```

Comment:
This is not a descriptive text for a link. Screenreader are able to collect a list of all links on your page and the user can then go through this list for a quick access. Now the user of your app would get a list of links where each time the screenreader announces `Click here!` for each link. So the user doesn't know what page is associated with which link. If you want it to be like this, then I would recommend adding an `aria-label` like this:
```
aria-label={props.website}
```